zenodo/base/templates/zenodo/faq.html

```diff
@@ -44,16 +40,16 @@
     </li>
     <li>
         <h5>Is my data safe with you / What will happen to my uploads in the unlikely event that {{config.CFG_SITE_NAME}} has to close?</h5>
-        <p>Yes, your data is stored in CERN Data Centre. Both data files and metadata is kept in multiple online replicas, as well as backed up to tape every night. CERN has considerable knowledge and experience in building and operating large scale digital repositories, and a commitment to maintain this data centre to collect and store 100s of PBs of LHC data as it grows over the next 20 years. In the highly unlikely event that {{config.CFG_SITE_NAME}} will have to close operations, we guarantee that we will migrate all content to other suitable repositories, and since all uploads have DOIs all citations and links to the dataset will not be affected.
+        <p>Yes, your data is stored in CERN Data Center. Both data files and metadata are kept in multiple online replicas and are backed up to tape every night. CERN has considerable knowledge and experience in building and operating large scale digital repositories and a commitment to maintain this data centre to collect and store 100s of PBs of LHC data as it grows over the next 20 years. In the highly unlikely event that {{config.CFG_SITE_NAME}} will have to close operations, we guarantee that we will migrate all content to other suitable repositories, and since all uploads have DOIs, all citations and links to {{config.CFG_SITE_NAME}} resources (such as your data) will not be affected.
         </p>
     </li>
     <li>
-        <h5>What does it cost / Can we pay for Zenodo services? </h5>
-        <p>Zenodo is free for the long tail of Science. In order to offer services to the more resource hungry research, we will introduce a ceiling to the free slice and offer paid for slices above, according to the business model developed within the sustainability plan. If you can't wait but immediately want to explore these paid for options, please <a href="/contact">contact us</a> and we will look at interim measures with you.</p>
+        <h5>What does it cost? /Can we pay for Zenodo services? </h5>
```

Comment:
I would keep it as-is for now, until the donation model has gone up. Then we can change it.